packages/trpc/server/routers/viewer/payments/chargeCard.handler.ts

```diff
@@ -79,9 +79,38 @@ export const chargeCardHandler = async ({ ctx, input }: ChargeCardHandlerOptions
     },
   };
 
+  let paymentCredentialUserId = ctx.user.id;
+
+  const eventType = booking.eventType;
+  const isTeamEvent = eventType?.schedulingType || eventType?.parentId;
+
+  if (isTeamEvent) {
```

Comment:
When charging an attendee's card using Stripe, we need to get the team owner's Stripe credentials